### PR TITLE
hardcode colors for world map

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.html [hunk 1]
```diff
@@ -70,80 +70,82 @@
...
           </mat-list-item>
         </mat-list>
-      </mat-card>
+      </article>
     </world-map>
     <article fxLayout fxLayoutAlign="end">
       <button mat-button color="primary" (click)="selectAll()">
         <mat-icon svgIcon="add"></mat-icon>
-        Select all available territories
+        <span>Select all available territories</span>
```
Comment (RemcoSimonides) [inline apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.html:148]: Why do you need `<span></span>`?

File: apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.scss [hunk 2]
```diff
@@ -18,18 +19,24 @@ p {
...
+    border-radius: 4px;
 
     .overlay {
       position: absolute;
-      height: 100%;
-      width: 100%;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
```
Comment (RemcoSimonides) [inline apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.scss:30]: This is doing the same thing, right?

File: apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.html [hunk 3]
```diff
@@ -70,80 +70,82 @@
...
       <ng-container *ngFor="let tag of (sold$ | async); trackBy: trackByTag">
-        <map-feature [tag]="tag.isoA3" color="background-disabled-button"
+        <map-feature [tag]="tag.isoA3" color="#9ca3c4"
           (mouseover)="displayTerritoryTooltip(tag.label, 'Rights sold')"
           (mouseout)="clearTerritoryTooltip()">
         </map-feature>
       </ng-container>
       <ng-container *ngFor="let tag of (selected$ | async); trackBy: trackByTag">
-        <map-feature [tag]="tag.isoA3" color="primary" weight="2"
+        <map-feature [tag]="tag.isoA3" color="#1a49f5"
```
Comment (RemcoSimonides) [inline apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.html:97]: No need for thicker border anymore?

**Review by RemcoSimonides — APPROVED** [review]

File: apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.html (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.html:148]: There is general style for button to have 8px between span & mat-icon

File: apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.scss (hunk 2 above)
Comment (GrandSchtroumpf) [reply apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.scss:30]: only if parent is block.

**Pull request merged**